charts/platform-system/templates/flux/cert-manager/namespace.yaml

```diff
@@ -0,0 +1,7 @@
+{{ if .Values.apps.certManager.enabled }}
```

Comment:
Can roll the `namespace.yaml` up to `.../templates/apps/cert-manager/namespace.yaml` since it's not implementation specific. I meant to do that to istio but it slipped my mind.

Reply:
moved